src/EditorFeatures/Test2/IntelliSense/CompletionRulesTests.vb

```diff
@@ -46,26 +45,24 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
         End Sub
 
         Private Sub TestMatches(v As String, wordsToMatch() As String)
-            Using New CultureContext("tr-TR")
-                Dim workspace = New TestWorkspace
-                Dim helper = CompletionHelper.GetHelper(workspace, LanguageNames.CSharp)
-                For Each word In wordsToMatch
-                    Dim item = CompletionItem.Create(word)
-                    Assert.True(helper.MatchesFilterText(item, v, CompletionTrigger.Default, Nothing), $"Expected item {word} does not match {v}")
-                Next
-            End Using
+            Dim culture = New CultureInfo("tr-TR", useUserOverride:=False)
+
+            Dim workspace = New TestWorkspace
+            Dim helper = CompletionHelper.GetHelper(workspace, LanguageNames.CSharp)
+            For Each word In wordsToMatch
+                Dim item = CompletionItem.Create(word)
+                Assert.True(helper.MatchesFilterText(item, v, culture), $"Expected item {word} does not match {v}")
+            Next
         End Sub
 
         Private Sub TestNotMatches(v As String, wordsToNotMatch() As String)
-            Using New CultureContext("tr-TR")
-                Dim workspace = New TestWorkspace
-                Dim helper = CompletionHelper.GetHelper(workspace, LanguageNames.CSharp)
-                For Each word In wordsToNotMatch
-                    Dim item = CompletionItem.Create(word)
-                    Assert.False(helper.MatchesFilterText(item, v, CompletionTrigger.Default, Nothing), $"Unexpected item {word} matches {v}")
-                Next
-            End Using
-
+            Dim culture = New CultureInfo("tr-TR", useUserOverride:=False)
```

Comment:
It's not a CultureContext.  It's just a CultureInfo.  Those don't need to be disposed.  I removed the code that tried to do all this work by setting thread local data.